stronk/schemas/user/mutations.py

```diff
@@ -21,9 +20,7 @@ class Arguments:
         currentProgram = graphene.Int(required=False)
 
     def mutate(root, info, name, username, email, currentProgram=None):
-        id = c.TEST_ID if current_app.config['ENV'] == 'testing' else g.id
```

Comment:
I think it's safe to remove the `TEST_ID` from the const since it's only used here